go/store/nbs/file_manifest_test.go

```diff
@@ -77,6 +79,7 @@ func TestFileManifestLoadIfExists(t *testing.T) {
 }
 
 func TestFileManifestLoadIfExistsHoldsLock(t *testing.T) {
+	t.Skip("TODO")
```

Comment:
this test fails now that we don't hold the lock while reading

Reply:
Remove it? Its name seems to imply it's testing for behavior we never intend to implement.